frontend/public/components/dashboard/dashboard.scss

```diff
@@ -0,0 +1,8 @@
+.co-dashboard-body {
+  background-color: $pf-color-black-200;
+  padding: 1rem;
+
+  .pf-l-grid {
```

Comment:
This doesn't follow BEM

Reply:
added new class which Im now passing to `Grid` component directly